src/app/containers/PageHandlers/withContexts.jsx

```diff
@@ -12,6 +13,7 @@ const WithContexts = Component => {
   const WithContextsContainer = props => {
     const {
       bbcOrigin,
+      data,
```

Comment:
Since we're not using all of `data`, we should get `const { status} = data;` and only pass `status` to the RequestContext.

Reply:
Done :) We descussed IRL about splitting data up to avoid this destructuring. I have elected to do that in a future issue https://github.com/bbc/simorgh/issues/3839, hope this is ok? :)

Reply:
Sure, it'd simplify changes in this PR. Thanks for raising the issue.